### DISCUSSION TITLE
Review Changes button does nothing in Windows Firefox

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#78680] [Pull Requests]
### Select Topic Area

Bug

### Body

As evidenced here:

https://github.com/community/community/assets/10906554/c0a6adf0-6da9-4c8e-bba6-f8c5e1b3dfc3

Also there are other drop-downs that similarly do nothing in Firefox (can't recall them off teh top of my headd)

**Comment by LauraS-OCC** [comment]
Also had this happen in Vivaldi browser. Our team is reporting it on multiple browsers. Refreshing the page (but not switching the browser, weirdly) is a workaround that is working for us.

**Comment by queenofcorgis** [comment]
Thanks for reporting - I've made sure this report has gotten passed along to our team 🤠 .

**Reply by LauraS-OCC** [reply]
Thanks!
